src/assets/components/ns-lockup/flat-media-group.html

```diff
@@ -0,0 +1,15 @@
+<ns-lockup type="flat">
+  <h2 slot="heading">Fingerstache flexitarian vaporware</h2>
+  <p slot="paragraph">Four loko vice ramps kickstarter next level venmo stumptown kogi salvia.</p>
+  <ns-cta slot="cta" href="#!">Jawn butcher</ns-cta>
+  <ns-media-group
+    slot="image"
+    type="bento"
+    media="[
+    {&quot;type&quot;:&quot;image&quot;,&quot;src&quot;:&quot;https://andijcdn.sirv.com/nds/testcard-4x5.png?w=720&quot;},
+    {&quot;type&quot;:&quot;image&quot;,&quot;src&quot;:&quot;https://andijcdn.sirv.com/nds/testcard-4x3.png?w=720&quot;},
+    {&quot;type&quot;:&quot;image&quot;,&quot;src&quot;:&quot;https://andijcdn.sirv.com/nds/testcard-4x3.png?w=720&quot;},
+    {&quot;type&quot;:&quot;image&quot;,&quot;src&quot;:&quot;https://andijcdn.sirv.com/nds/testcard-4x5.png?w=720&quot;}
```

Comment:
When the asset has been added, change to:

```suggestion
    {&quot;type&quot;:&quot;image&quot;,&quot;src&quot;:&quot;https://nucleus.design/placeholder-4x5-360x450px.webp&quot;},
    {&quot;type&quot;:&quot;image&quot;,&quot;src&quot;:&quot;https://nucleus.design/placeholder-4x3-480x360px.webp?w=720&quot;},
    {&quot;type&quot;:&quot;image&quot;,&quot;src&quot;:&quot;https://nucleus.design/placeholder-4x3-480x360px.webp&quot;},
    {&quot;type&quot;:&quot;image&quot;,&quot;src&quot;:&quot;https://nucleus.design/placeholder-4x5-360x450px.webp&quot;}
```